services/pinpoint/src/it/java/software/amazon/awssdk/services/pinpoint/PinpointIntegTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.pinpoint;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.pinpoint.model.CreateAppResponse;
+import software.amazon.awssdk.testutils.service.AwsTestBase;
+
+public class PinpointIntegTest extends AwsTestBase {
+
+    private static final String APP_NAME = "PinpointIntegTest" + +System.currentTimeMillis();
+
+    protected static PinpointAsyncClient pinpointAsyncClient;
+
+    private static String appId;
+
+    @BeforeClass
+    public static void setup() {
+        pinpointAsyncClient = PinpointAsyncClient.builder()
+                                                 .region(Region.US_WEST_2)
+                                                 .credentialsProvider(CREDENTIALS_PROVIDER_CHAIN)
+                                                 .build();
+
+        CreateAppResponse response = pinpointAsyncClient.createApp(b -> b.createApplicationRequest(r -> r.name(APP_NAME))).join();
+
+        appId = response.applicationResponse().id();
```

Comment:
Is it necessary to create an app? Would doing a list operation be sufficient (to avoid creating any resources)?

Reply:
Nope, updated to use `getApps` instead